.gitignore

```diff
@@ -12,6 +12,14 @@
 **/bin
 *~
 
+# Vaadin auto-generated files
+appsec-kit-demo/frontend/generated
```

Comment:
`1.0` branch doesn't have `appsec-kit-demo`, it contains four demos.

Reply:
Those demos don't contain any of these files or folders as they are for V7 and V8 versions. I these changes are not needed in 1.0 branch.